src/client/interpreter/locators/services/cacheableLocatorService.ts

```diff
@@ -48,16 +49,20 @@ export abstract class CacheableLocatorService implements IInterpreterLocatorServ
             this.addHandlersForInterpreterWatchers(cacheKey, resource)
                 .ignoreErrors();
 
-            const promise = this.getInterpretersImplementation(resource)
+            const stopWatch = new StopWatch();
+            this.getInterpretersImplementation(resource)
                 .then(async items => {
                     await this.cacheInterpreters(items, resource);
                     traceVerbose(`Interpreters returned by ${this.name} are of count ${Array.isArray(items) ? items.length : 0}`);
                     traceVerbose(`Interpreters returned by ${this.name} are ${JSON.stringify(items)}`);
+                    sendTelemetryEvent(EventName.PYTHON_INTERPRETER_DISCOVERY, stopWatch.elapsedTime, { locator: this.name, counter: Array.isArray(items) ? items.length : 0 });
```

Comment:
```suggestion
                    sendTelemetryEvent(EventName.PYTHON_INTERPRETER_DISCOVERY, stopWatch.elapsedTime, { locator: this.name, interpreters: Array.isArray(items) ? items.length : 0 });
```

Counter is not a meaningful name (its used in for loops).

Reply:
Please fix this before merging.